mteb/abstasks/TaskMetadata.py

```diff
@@ -477,3 +477,95 @@ def __hash__(self) -> int:
     @property
     def revision(self) -> str:
         return self.dataset["revision"]
+
+    def create_dataset_card_data(self) -> tuple[DatasetCardData, dict[str, str]]:
+        """Create a DatasetCardData object from the task metadata.
+
+        Returns:
+            A DatasetCardData object with the metadata for the task with kwargs to card
+        """
+        # todo figure out datasets with multiple types. E. g. one dataset as classification and ZeroShotClassification
+        mteb_task_type_to_datasets = {
+            # Text
+            "BitextMining": "translation",
+            "Classification": "text-classification",
+            "MultilabelClassification": "text-classification",
+            "Clustering": "zero-shot-classification",  # not sure
```

Comment:
I would just put "text-clustering" same for images if we can have "non-real" types